website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,187 @@
+[ResourceQuota][k8s-resources-quotas] configuration in a namespace will cause all pods without 
+[Requests or limits][k8s-requests-limits] for `CPU` and `Memory` to be rejected.
+
+Cluster administrators will naturally want to provide enough resource requests to their Operators in order to give them 
+sufficient memory resources for safe operations, and avoid that other workloads impact them negatively.
+For this reason, if you are integrating with [OLM][olm], you should, **always ship your operator `Deployment` 
+with resource requests amounts for CPU and memory**.
+
+The [ResourceQuota][k8s-resources-quotas] configuration also is applied to tenant workloads that are managed by your operator. Cluster administrators will typically set
+a [ResourceQuota][k8s-resources-quotas] for each tenant's namespace as part of the onBoarding. If your operator creates `Pods/Deployments` inside
+the tenant namespace which do not specify at least resource requests for `CPU` and `Memory`, those `Pods/Deployments` may fail to schedule and tenants
+may not be able to use your operator. Therefore and for ensuring a suitable scheduling your operator **always apply at least resource requests for CPU and
+memory** for `Pods/Deployments` that it creates as part of the reconciliation logic. Ideally your operator also applies 
+memory limits to those `Pods/Deployments`. You may also consider CPU limits.
+
+#### General guidelines
+
+**Note that pretty much all controllers consume:**
+- `CPU`: largely based on the number of reconciliations they perform, which are generally related to event activity for resources they're watching.
+- `Memory`: largely based on the number of primary resources that exist (multiplied by some factor based on the number of operand resources they need to watch as a result) via informer caches.
+
+Operator authors cannot anticipate how they scale in usage in all customer environments. So, a single configuration that 
+fits well for all scenarios is not possible then, clusters administrators must be allowed to change the values 
+in order to respond to changes in their clusters needs, such as by resizing control plane nodes to provide more resources.
+
+**Following are some general recommendations to manage the resources:**
+
+- **MUST** declare resource requests for both CPU and memory for the `Operator/manager`  and any `Pod/Deployment` managed by it
+- **OPTIONALLY** setting the resources limit for Memory and CPU for the Operator Pod and any `Pod/Deployment` managed by it.
+**NOTE** If your project is integrated with OLM then, admins would be able to customize it for the `Operator/manager` 
+via the [subscription][olm-subscriptions]). Also, be aware that if the project was generated by Kubebuilder or SDK scaffold 
+then some values for the `Operator/manager`(See `config/manager/manager.yaml`) 
+are populated by default to get you started, however, you ought to optimize them based on your own tests and the specific needs of your operator.
```

Comment:
That seems the same : https://github.com/operator-framework/operator-sdk/pull/5353#discussion_r753660008

Also, since you say that you are ok with I am closing it as sorted out. : -)